### PR TITLE
Chain specific ballots for V2

### PR DESCRIPTION
**Pull request by aeolianeth**
## What does this PR do and why?

Closes https://github.com/jbx-protocol/juice-interface/issues/826

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the
changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

File: src/constants/v2/ballotStrategies/index.ts [hunk 1]
```diff
@@ -1,24 +1,38 @@
...
 import { t } from '@lingui/macro'
 
+import { readNetwork } from 'constants/networks'
+
+const BALLOT_ADDRESSES: { [k: string]: { [j: string]: string } } = {
+  THREE_DAY: {
+    rinkeby: '0xcA65D0348E6d53BD29d52e298140375cbC6d3C0D',
+    mainnet: '0x5eaBF1D7A8c6942534cCb3489Ff221b2CBc0255b',
+  },
+  // TODO waiting on 7-day buffer contract to be deployed.
```
Comment (aeolianeth) [inline src/constants/v2/ballotStrategies/index.ts:11]: note

File: src/utils/v2/__tests__/fundingCycle.test.ts [hunk 2]
```diff
@@ -10,6 +10,8 @@ import { invertPermyriad } from 'utils/bigNumbers'
 
 import { decodeV2FundingCycleMetadata } from '../fundingCycle'
 
+process.env.REACT_APP_INFURA_NETWORK = 'mainnet'
```
Comment (aeolianeth) [inline src/utils/v2/__tests__/fundingCycle.test.ts:13]: fixes test

File: src/components/v2/V2Project/V2FundingCycleSection/PayoutSplitsCard.tsx [hunk 3]
```diff
@@ -34,42 +34,46 @@ export default function PayoutSplitsCard({
...
     balanceInDistributionLimitCurrency,
     isPreviewMode,
+    loading,
   } = useContext(V2ProjectContext)
   const ETHPaymentTerminalFee = useETHPaymentTerminalFee()
 
   const [distributePayoutsModalVisible, setDistributePayoutsModalVisible] =
     useState<boolean>()
 
+  const isLoadingStats =
```
Comment (aeolianeth) [inline src/components/v2/V2Project/V2FundingCycleSection/PayoutSplitsCard.tsx:44]: fixes dodgy skeleton loader in project preview

File: src/components/shared/IssueTicketsButton.tsx [hunk 4]
```diff
@@ -75,6 +79,15 @@ export default function IssueTicketsButton({
     )
   }
 
+  const onClose = () => {
+    setModalVisible(false)
+
+    // remove newDeploy=true query parameter
+    history.replace({
```
Comment (aeolianeth) [inline src/components/shared/IssueTicketsButton.tsx:86]: removes the param when closing the modal, so subsequent refreshes of the page don'e trigger the modal

File: src/components/Projects/index.tsx [hunk 5]
```diff
@@ -138,8 +141,8 @@ export default function Projects() {
             <Trans>Projects on Juicebox</Trans>
           </h1>
 
-          <Link to="/create">
-            <Button>
+          <Link to={v2Enabled ? '/v2/create' : '/create'}>
```
Comment (aeolianeth) [inline src/components/Projects/index.tsx:144]: Bug identified by @wraeth-eth , fixed here.

Comment (johnnyd-eth) [reply src/components/Projects/index.tsx:144]: Nice

**Comment by johnnyd-eth** [comment]
Fuck yeah @aeolianeth. Code all looks great to me and worth getting merged asap

**Review by johnnyd-eth — APPROVED** [review]

**Pull request merged**